harness/buildString.js

```diff
@@ -14,3 +14,19 @@ function buildString({ loneCodePoints, ranges }) {
   }
   return result;
 }
+
+function test(regex, string, expression) {
+  if (!regex.test(string)) {
+    for (const symbol of string) {
+      const hex = symbol
+        .codePointAt(0)
+        .toString(16)
+        .toUpperCase()
+        .padStart(6, "0");
```

Comment:
... I want to say yes as we'll release it in the 2017 version and it's well implemented in all the major active browsers.

Let's ship it and we can fix it if we break someone else's implementation. The chances are minimal.